### PR TITLE
feat: authentication pages added

### PR DESCRIPTION
**Pull request by rdshinde**
- Signup page added
- Login page added

Screenshot 2022-02-18 at 5 43 27 PM

Screenshot 2022-02-18 at 5 47 23 PM

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- Local Stylesheet -->
+    <link rel="stylesheet" href="../style.css" />
+    <title>Swadeshi</title>
```
Comment (Sameony) [inline templates/login.html]: you can give a more specific title to the page such as login | swadeshi

**Review by Sameony — APPROVED** [review]
Good code
a minor suggestion regarding the titles of the webpages:

File: templates/login.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- FontAwesome CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
```
Comment (kjais1720) [inline templates/login.html]: you can import this cdn in a common CSS file (that you will import in every page, like main.css etc.), then you won't need to import the same link in every page and the structure would look much clean

**Review by kjais1720 — APPROVED** [review]
Good use of utility classes and reusable components, just a small suggestion ( as mentioned in the comment ).

File: templates/login.html (hunk 1 above)
Comment (rdshinde) [reply templates/login.html]: Ya I will change it.

File: templates/login.html (hunk 2 above)
Comment (rdshinde) [reply templates/login.html]: Ya got it thank you. 🙏

**Pull request merged**